src/cwe_checker_lib/src/intermediate_representation/program.rs

```diff
@@ -48,5 +48,49 @@ mod tests {
                 address_base_offset: 0,
             }
         }
+        /// Returns Program with malloc, free and other_function
+        pub fn mock() -> Program {
+            let malloc = ExternSymbol {
+                tid: Tid::new("malloc"),
+                addresses: vec![],
+                name: "malloc".to_string(),
+                calling_convention: None,
+                parameters: vec![],
+                return_values: vec![],
+                no_return: false,
+                has_var_args: false,
+            };
+            let free = ExternSymbol {
+                tid: Tid::new("free"),
+                addresses: vec![],
+                name: "free".to_string(),
+                calling_convention: None,
+                parameters: vec![],
+                return_values: vec![],
+                no_return: false,
+                has_var_args: false,
+            };
```

Comment:
malloc and free should have parameters and return values as expected from the corresponding libc functions in the mocked program. This probably means that we need 2 versions, one for x64 and one for ARM, because of the different param/return registers.